.github/workflows/go.yml

```
@@ -0,0 +1,23 @@
+name: Go
+
+on: [push]
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3.0.2
```

[Reference](https://github.com/kohofinancial/documentation/blob/main/best%20practices/github-actions.md#version-pinning-and-dependency-management)